src/import/engine.rs

```diff
@@ -63,7 +63,9 @@ impl Engine {
         let commit_ts = batch.get_commit_ts();
         for m in batch.take_mutations().iter_mut() {
             let k = Key::from_raw(m.get_key()).append_ts(commit_ts);
-            wb.put(k.encoded(), m.get_value()).unwrap();
+            match m.get_op() {
```

Comment:
What's the point?

Reply:
Call `wb.put` without check `m.get_op()` is a bad idea because if we add new enum entries (e.g. Delete) later, we can't do same thing with `Put`.

Reply:
But this doesn't solve the problem. Compatibility issue still exists.